### ISSUE TITLE
Large control Tower Won’t load into Standard Industrial or Advanced Industrial cargo

### ISSUE THREAD

**Issue by OperationDx** [EvEmu-Project/evemu_Crucible#266]
**Describe the bug**
Bug - Large control Tower Won’t load into Standard Industrial or Advanced Industrial cargo - Ships only needs 8000m3 cargo space - Amarr / Gallente / Caldari / Minmatar are effected

**To Reproduce**
Buy and assemble any factions hauler / industrial ship or Advanced
Fit ship to hold at least 8000m3 cargo space.
Purchase any large control tower on the market.
Attempt to load the tower into the ships cargo hold.

**Expected behavior**
Any faction's industrial ship with enough cargo space should be able to load and deploy a control tower of any size. But this bug won't even allow the player to load a control tower into the cargo hold.

**Screenshots**
May add later.

**System Details (please complete the following information):**
 - Server OS: [Ubuntu]
 - Docker - Latest Staging build as of 08/07/23

**Additional context**
Nothing extra to report.

**Comment by OperationDx** [comment]
![Cargo_Issue](https://github.com/EvEmu-Project/evemu_Crucible/assets/141080303/bd67f015-7ad6-4cb9-b049-5ec87d935edd)

Cargo space is not correctly calculated when fitting cargo expanders and cargo rig. Game thinks cargo space is much less then it is actually. This seems to be an issue with all ships.

**Comment by OperationDx** [comment]
Makes sense why the game won't allow a control tower in the cargo hold if it thinks it's maxed out at 4750m3 when it should be 19,027 with this setup.

**Comment by OperationDx** [comment]
![2023 08 07 23 02 00](https://github.com/EvEmu-Project/evemu_Crucible/assets/141080303/d708c683-4267-4a9c-812b-db9978cbbb58)

**Comment by DeerHunt3r** [comment]
Fixed it with a small change to the GenericModule.cpp file.  Change this line -  

 if (m_shipRef->GetPilot()->IsDocked() and (!m_shipRef->IsUndocking())) {
        m_ModuleState = Module::State::Online;
        SetAttribute(AttrOnline, EvilOne, !m_shipRef->GetPilot()->IsLogin());
        return;
    }

To this:
 if (m_shipRef->GetPilot()->IsDocked()
        && !m_shipRef->IsUndocking()
        && m_shipRef->GetPilot()->IsLogin())
    {
        m_ModuleState = Module::State::Online;
        SetAttribute(AttrOnline, EvilOne, !m_shipRef->GetPilot()->IsLogin());
        return;
    }

Once I learn how to create a pr and push things from my repo I'll do it the right way.

**Comment by DeerHunt3r** [comment]
Ugh, hang on ran into another issue with it..... testing still.
